### ISSUE TITLE
Main void repo

### ISSUE THREAD

**Issue by Encoded14** [Makrennel/hyprland-void#79]
Couldnt this be in the main void repos now that hyprland uses aquamarine?

**Comment by Makrennel** [comment]
> Couldnt this be in the main void repos now that hyprland uses aquamarine?

No. Void rejects Hyprland on ideological grounds, and Void is too chronically out of date to ever hope to have a quickly developing package in its repositories as Hyprland currently is – which is also why this repo has still not been updated to the latest version, as it simply cannot be built on Void.

**Comment by soanvig** [comment]
https://github.com/Makrennel/hyprland-void/issues/59

**Comment by Encoded14** [comment]
im confused, hyprland doesnt even use wl-roots anymore so why does it matter?

**Comment by soanvig** [comment]
> im confused, hyprland doesnt even use wl-roots anymore so why does it matter?

In that case ask void-packages maintainers: https://github.com/void-linux/void-packages/issues/50824
They say: nope
